### PR TITLE
Fix / old password form error after validation

### PR DESCRIPTION
**Pull request by MelissaDTH**
## This PR

- Adds a missing translation for when the old password is not correct after submitting the 'Password reset' form (I've used the error message obtained from the API)
- Updates the password fields to include the `autocomplete` attribute to improve accessibility
- Solves: OTT-1071

### CODE REVIEW

**Comment by royschut** [comment]
You're now trying to match the filled in old password with the new password. That's never what we want, right? The user wants to change it. 
We should let the backend validate the old password and just present the api error. You also don't need the translation then.
Edit: last statement was incorrect, you still need the translation. :)

**Comment by MelissaDTH** [comment]
> You're now trying to match the filled in old password with the new password. That's never what we want, right? The user wants to change it. We should let the backend validate the old password and just present the api error. You also don't need the translation then.

Good point regarding the api error. I looked at the way it is currently done with the validation between the 2 new passwords. I will go back to the drawing board!

**Update:** discussed with Roy that we do need the translations in order to display the error in Spanish as well. We do now however take the 'old password does not match' api error in account and show it accordingly!

File: platforms/web/public/locales/es/account.json [hunk 1]
```diff
@@ -204,7 +204,8 @@
     "password_reset_success": "Contraseña restablecida",
     "password_reset_text": "",
     "password_strength": "Aunque tu contraseña cumple con los criterios de una contraseña segura, incluye elementos que no son recomendables, como frases prohibidas o palabras repetidas. Por favor, modifícala para garantizar la seguridad de tu cuenta.",
-    "passwords_do_not_match": "Las contraseñas no coinciden",
+    "passwords_do_not_match": "La contraseña antigua no coincide",
```
Comment (royschut) [inline platforms/web/public/locales/es/account.json]: I think this is a mistake, this translation should not be updated.

**Review by royschut — APPROVED** [review]
Approved, provided you undo the "passwords_do_not_match" update, and put its value into "old_password_does_not_match".

File: platforms/web/public/locales/es/account.json (hunk 1 above)
Comment (royschut) [reply platforms/web/public/locales/es/account.json]: But it's the value for "old_password_does_not_match" actually.

Comment (MelissaDTH) [reply platforms/web/public/locales/es/account.json]: Sharp, has been fixed!

**Review by langemike — APPROVED** [review]
Nice!

Interesting to see that we missed the `autocomplete` attribute previously for the password field, but these thing can happen ;)

**Review by royschut — APPROVED** [review]
Nice!

**Pull request merged**